hddm/models/hddm_info.py

```diff
@@ -102,10 +102,10 @@ class HDDM(HDDMBase):
     def __init__(self, *args, **kwargs):
         self.slice_widths = {'a':1, 't':0.01, 'a_std': 1, 't_std': 0.15, 'sz': 1.1, 'v': 1.5,
                              'st': 0.1, 'sv': 3, 'z_trans': 0.2, 'z': 0.1,
-                             'p_outlier':1., 'v_std': 1,'alpha':3.5,'dual_alpha':3.5}
+                             'p_outlier':1., 'v_std': 1, 'alpha':3.5, 'dual_alpha':3.5}
```

Comment:
Are you sure that this is the right slice_widths? This depends on the expected width of the posterior, and 3.5 is very wide.

Reply:
Thanks for catching this. I've now set them to be 1.5,  same value as v.